pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -558,6 +558,16 @@ type TiKVSpec struct {
 	// EnableNamedStatusPort enables status port(20180) in the Pod spec.
 	// If you set it to `true` for an existing cluster, the TiKV cluster will be rolling updated.
 	EnableNamedStatusPort bool `json:"enableNamedStatusPort,omitempty"`
+
+	// ScaleInParallelism configures max scale in replicas for TiKV stores.
+	// Defaults to 1
+	// +optional
+	ScaleInParallelism *int32 `json:"scaleInParallelism,omitempty"`
```

Comment:
Use `// +kubebuilder:default=1` to add the default value?

Reply:
Get.